src/main/resources/db/migration/mysql/V1.6__organisational-unit.sql

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS `organisational_unit`;
+
+CREATE TABLE `organisational_unit` (
+  `id` smallint(5) unsigned NOT NULL AUTO_INCREMENT,
+  `parent_id` smallint(5) unsigned DEFAULT NULL,
+  `code` varchar(10) NOT NULL,
```

Comment:
Length is 20in the entity Mapping. I would say decrease the entity mapping to 10